pkg/shell/shell.go

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
Hmm I do wonder if this should move into `cilium/hive` instead.

Reply:
Honestly - I was surprised to see that `cilium/hive` already provides built-in shell commands. Even though it makes sense to interactively play with the hive cell - i wonder whether we should add even more "Cilium related" "runtime" aspects to this repo - as it's actual "just" provides a dependency injection framework :thinking: